packages/taco/integration-test/encrypt-decrypt.test.ts

```diff
@@ -25,31 +28,44 @@ const DOMAIN = 'lynx';
 const RITUAL_ID = 27;
 const CHAIN_ID = 80002;
 
-describe.skipIf(!process.env.RUNNING_IN_CI)(
+describe.skipIf(!process.env.RUNNING_IN_CI).each<[string, any, any, any]>([
+  [
+    'ethers',
+    new ethers.providers.JsonRpcProvider(RPC_PROVIDER_URL),
+    new ethers.Wallet(ENCRYPTOR_PRIVATE_KEY),
+    new ethers.Wallet(CONSUMER_PRIVATE_KEY),
+  ],
+  [
+    'viem',
+    createPublicClient({
+      chain: polygonAmoy,
+      transport: http(RPC_PROVIDER_URL),
+    }),
+    privateKeyToAccount(ENCRYPTOR_PRIVATE_KEY),
+    privateKeyToAccount(CONSUMER_PRIVATE_KEY),
+  ],
+])(
```

Comment:
In some other comments, I was referring to those `any` here.
```suggestion
describe
  .skipIf(!process.env.RUNNING_IN_CI)
  .each<
    | ['ethers', ethers.providers.Provider, ethers.Wallet, ethers.Wallet]
    | ['viem', PublicClient, Account, Account]
  >([
    [
      'ethers',
      new ethers.providers.JsonRpcProvider(RPC_PROVIDER_URL),
      new ethers.Wallet(ENCRYPTOR_PRIVATE_KEY),
      new ethers.Wallet(CONSUMER_PRIVATE_KEY),
    ],
    [
      'viem',
      createPublicClient({
        chain: polygonAmoy,
        transport: http(RPC_PROVIDER_URL),
      }),
      privateKeyToAccount(ENCRYPTOR_PRIVATE_KEY),
      privateKeyToAccount(CONSUMER_PRIVATE_KEY),
    ],
  ])(
```
Sorry I missed posting this with the previous comments.

Reply:
✔️